### ISSUE TITLE
Delta-Interactive Sync will ban number even on update node

### ISSUE THREAD

**Issue by Sdlion** [mgp25/Chat-API#1969]
- [x] You are using PHP > 5.6
- [x] You have installed all extensions and modules listed in [here](https://github.com/mgp25/Chat-API/wiki/Dependencies). PHP Protobuf and Curve25519 are required!
- [x] You have checked if someone has already asked the same issue you have.
- [x] You have read the [wiki](https://github.com/mgp25/Chat-API/wiki).
- [x] Is not a programming question.
- [x] You are using latest Chat API code.

### Error

Situation: 
First login using the recommended procedure, adding 1 number. 
The scripts detects a "number-update" event node and Syncs with defaults (delta-interactive), unsubscribes and subscribes.

The main script is run, and a update-node is received from the contact that was added on the first login. Then the connection is closed without shutdown message and is banned without even received unsubscription subscription responses. Log in again shows that the number is banned.

The debug log in the "debug log" section is **from this session** (without the first login output).

This is the third time I see a number of mine banned after a delta-interactive sync. 
In this last version I only sync (delta interactive) all my added contacts after the first login on the script (and if it has been past 1 hour at least since last sync.... and btw, wait 6 hours before resyncing contacts won't do the trick either).

The first and second time I would resync on `onLoginSuccess()` if it have been past 6 hours (for the former) or 1 hour (for the later try) since the last sync of any contact. An update-contact notification node would trigger a sync-unsubscribe-subscribe procedure and would update that timestamp too, so it would need to have pass 6 or 1 hour since that action too. This is the debug log from the 1hour wait version http://pastebin.com/H9YRUHSm , banned without messages being received or send for 3 hours at least, stable loop after line 3,000, fail after line 4,150. I think that debug log is as important as this one, but this is simpler.

Since my script won't send messages unless it has received a message first, I decided to move out the contacts sync procedure out of `onLoginSuccess()` to the construct method of my class, checking anyway that it has been 1 hour at least since a contact has been synced. But is still resyncing on an update-contact notification node. The debug log below is from this script version I mention, note that this was the second sync that the number has made, being the first the full sync of the first login.

### Debug log

```xml
tx  <stream:features></stream:features>

tx  <auth user="xxxxxxxxxxx49" mechanism="WAUTH-2"></auth>

rx  <from s.whatsapp.net=""></from>

rx  <stream:features></stream:features>

rx  <challenge>e7ef2c9906ad234abce8f2b9687aa8c8645aa70e</challenge>

tx  <response>6bd465aa715c1702a4c52d347f459631c2eb2900128b8c764dbfbe4588aeabc8659f456497887dcecf5cdf2de8861421c722baf7e82a6ada62c2f96304b4ab53b4821a328f71cdaacedc2e0932e9d7</response>

rx  <success t="1465052314" props="5" kind="free" status="active" creation="1465003252" expiration="4444444444">7110c161c0f3867f13e980552a30c34f7b86d63d</success>

tx  <presence name="Tests" type="available"></presence>

tx  <iq id="01" xmlns="jabber:iq:privacy" type="get">
tx    <query>
tx      <list name="default"></list>
tx    </query>
tx  </iq>

tx  <iq id="02" xmlns="urn:xmpp:whatsapp:push" type="get" to="s.whatsapp.net">
tx    <config></config>
tx  </iq>

tx  <iq id="03" to="s.whatsapp.net" type="set" xmlns="encrypt">
tx    <identity>a00ea18a8ccdc71228feb63d51cffdb51ee44f37bc9a7c9e897b22051eb01747</identity>
tx    <registration>7fffffff</registration>
tx    <type>05</type>
tx    <list>
tx      <key>
tx        <id>3c3182</id>
tx        <value>f89a834f11198aa67f5767e2bd3afa9cd856531578a191b39c2dc41cf6d84911</value>
tx      </key>
tx      <key>
tx        <id>3c3183</id>
tx        <value>d4c8fdc7ec5ec0f55fec9c83fa9218dac23834b389b6be730afbffe9c1088d72</value>
tx      </key>
tx      <key>
tx        <id>3c3184</id>
tx        <value>c97ff760cdba787244d4443158f71c7a17dbae5604d363e316c7373141bcb90a</value>
tx      </key>
tx      <key>
tx        <id>3c3185</id>
tx        <value>da3e753ab2c5f768249b1b9135bb7fc7a8577b36070c7603255f448ae91a7b18</value>
tx      </key>
tx      <key>
tx        <id>3c3186</id>
tx        <value>14948bc92ae906eb949e79a34156bc1a1410c8b5ef34a94cbe8261eeebdb2c38</value>
tx      </key>
tx      <key>
tx        <id>3c3187</id>
tx        <value>45d996509c2139a2f1b6ce0eb8c38bfdcbf6fafa3163751f12ee46555137c050</value>
tx      </key>
tx      <key>
tx        <id>3c3188</id>
tx        <value>50272f77058a2afd8137c36bab29ea1f28b129b756350bc84d598420c08cb140</value>
tx      </key>
tx      <key>
tx        <id>3c3189</id>
tx        <value>aa70ee3d06a1df3e06f52cc78225971ea6d5511ec17664d2c1fedc4286c8c711</value>
tx      </key>
tx      <key>
tx        <id>3c318a</id>
tx        <value>b744f79187730071d6e488bc859643aec407955794fbfe96199811b9797bbb66</value>
tx      </key>
tx      <key>
tx        <id>3c318b</id>
tx        <value>1c5af12966723cadcbb36d89232fb851cd6cb7806784b7f215428067763f8d0c</value>
tx      </key>
tx      <key>
tx        <id>3c318c</id>
tx        <value>b0db8d15f1ba93ad43946d308e7c9f32e024c77adfb40ffafb18fa3edf7c5e22</value>
tx      </key>
tx      <key>
tx        <id>3c318d</id>
tx        <value>03bd7ba244c88deeebe21e11ef178a394219424b144f35117f27e6cc627fb71b</value>
tx      </key>
tx      <key>
tx        <id>3c318e</id>
tx        <value>05c1645c1f777d469e59ce3d74b6d1f87e5b5e67f8742a984e316fc944fcb354</value>
tx      </key>
tx      <key>
tx        <id>3c318f</id>
tx        <value>f2de42b81593d9b8b4e644b499ac285f5a3cc8b3345c568f39340117ea9c5540</value>
tx      </key>
tx      <key>
tx        <id>3c3190</id>
tx        <value>8efad064375979795e6b31683893233e94d12c8b83573ce2a3ad43f52cb8114c</value>
tx      </key>
tx      <key>
tx        <id>3c3191</id>
tx        <value>ff9e127464d338d1b5fca37f3f3641e7c79dacf403612c22e0b2e68ec9d93402</value>
tx      </key>
tx      <key>
tx        <id>3c3192</id>
tx        <value>3e46efa59214f9b591673e7d4aba44080d2b0a40f9363e10ca2f9d8eda695a39</value>
tx      </key>
tx      <key>
tx        <id>3c3193</id>
tx        <value>5db037f318f7c56895a2ed754cac627fc3355d181e32b835e2054ef8af62d07d</value>
tx      </key>
tx      <key>
tx        <id>3c3194</id>
tx        <value>b8e659347a98dccd2dcaeec08903b741642b93be20832650245b34e15699c607</value>
tx      </key>
tx      <key>
tx        <id>3c3195</id>
tx        <value>3c635d956c386c1312757df1202ec148faf2014062097ff310e406d4b5057965</value>
tx      </key>
tx      <key>
tx        <id>3c3196</id>
tx        <value>2db6b6256955422ea3c598e68e7ff7424eb6a67b3f9ac2d743f3d1130705f909</value>
tx      </key>
tx      <key>
tx        <id>3c3197</id>
tx        <value>d23724acb977d47f45eb7c0b73502e2ea011a21ce0c560fa7ece032f70623773</value>
tx      </key>
tx      <key>
tx        <id>3c3198</id>
tx        <value>93de037189f617369f9b65b132df3b6dd871fd0d772d2a93f2f79a564e461d21</value>
tx      </key>
tx      <key>
tx        <id>3c3199</id>
tx        <value>2ff58fffa4848edaf3aa658daf762f36f100b051c3db6a333e55d8abd604ff15</value>
tx      </key>
tx      <key>
tx        <id>3c319a</id>
tx        <value>8c72dc72e3c19023a80a455e55a9a34de52bb8d41a6871ed360da5fd07b05b59</value>
tx      </key>
tx      <key>
tx        <id>3c319b</id>
tx        <value>0b2a961cce699d006fa72684dd657ecd88d22a00ce66d9fcc5dd21f7f10b3338</value>
tx      </key>
tx      <key>
tx        <id>3c319c</id>
tx        <value>20cdc57b6adcad4f358161fda733de0ce5bab731498cb6cd90c97a76c721b61e</value>
tx      </key>
tx      <key>
tx        <id>3c319d</id>
tx        <value>e61f50a876e232274ad743299b73e3d3a1af47159ce3050b4d4fdaf2598f5879</value>
tx      </key>
tx      <key>
tx        <id>3c319e</id>
tx        <value>451e5de63dd90b6e8b64b1de9fb671db7df6eb3ef822f8070abb82477e7c5130</value>
tx      </key>
tx      <key>
tx        <id>3c319f</id>
tx        <value>5cb554deb89548c51aca93e4cfc95e22bd4d0939c50ad3e530dfee3b1dbe5d39</value>
tx      </key>
tx      <key>
tx        <id>3c31a0</id>
tx        <value>6769f151ef1e2dc1c5236b615e40e5802f725965dcd6b2676121f03183135b00</value>
tx      </key>
tx      <key>
tx        <id>3c31a1</id>
tx        <value>ac6c64644ae9000bc69964df36da91e4bb981fe7e44b09594caae9d8c895c406</value>
tx      </key>
tx      <key>
tx        <id>3c31a2</id>
tx        <value>a0684f95c8d370c917ee059333ab94af2fb6ed674b8f8f1648534d60969a9f17</value>
tx      </key>
tx      <key>
tx        <id>3c31a3</id>
tx        <value>542650111b03e6e2f77a7ce2c8f16159d9aa31edf0c8b3214546cf2bdc4f0524</value>
tx      </key>
tx      <key>
tx        <id>3c31a4</id>
tx        <value>99de75ac6595df3cae4f4f29c9f93c55bc1aa545e6b13c514901df66219c0b01</value>
tx      </key>
tx      <key>
tx        <id>3c31a5</id>
tx        <value>e52552dc95a9f546306c9d4d8032e81ed97dd309ad82321cb2a4dd84fec97108</value>
tx      </key>
tx      <key>
tx        <id>3c31a6</id>
tx        <value>6e61884570a8ecc13f7e1541114f487bec7ccf38d0dbbd76b1a9a8541f24be60</value>
tx      </key>
tx      <key>
tx        <id>3c31a7</id>
tx        <value>f324d86bd32b7d8869d665267fb2875c83ee6deb253ce849e7c65bf024a73966</value>
tx      </key>
tx      <key>
tx        <id>3c31a8</id>
tx        <value>f2f789e6d557a5ed8637021fab01326d9816baccd8738415c05cbcdee4718e76</value>
tx      </key>
tx      <key>
tx        <id>3c31a9</id>
tx        <value>343d0d96d0d017160ff571b761310d0b57aaf498e814031fb68843b94459ee00</value>
tx      </key>
tx      <key>
tx        <id>3c31aa</id>
tx        <value>799a5a0291b8aeb8dc3d392444d5d87c6c6c2737bbfc7440ee8ef7609be0821e</value>
tx      </key>
tx      <key>
tx        <id>3c31ab</id>
tx        <value>1bfaa56ff70cf07bbb8d7c70b61363cc2c7f1790e132aeae9b989ae098715c48</value>
tx      </key>
tx      <key>
tx        <id>3c31ac</id>
tx        <value>238b7cb2c6aee7ffd5e99db8ac1bdf14ab21fe8519c8585619b2e5c721983430</value>
tx      </key>
tx      <key>
tx        <id>3c31ad</id>
tx        <value>cc3593f7a694b3e8da70e9bf2022a6754ad7c35f256c27ac06b5fac8a8592866</value>
tx      </key>
tx      <key>
tx        <id>3c31ae</id>
tx        <value>5a02d886dd4cec264af87e13ef960f3f30152e60b77cdfc6ea4af34fcea71715</value>
tx      </key>
tx      <key>
tx        <id>3c31af</id>
tx        <value>5bb5fa7c31b813c52c147847f682a10c6814be5b9dfa8299657ba7eb2b0df159</value>
tx      </key>
tx      <key>
tx        <id>3c31b0</id>
tx        <value>2aa6a69efc07a632fbbd67a50fa1b43cf368581ee431de08b7ccc969f5ebbe0d</value>
tx      </key>
tx      <key>
tx        <id>3c31b1</id>
tx        <value>b09fa2f2123acc1bb52adfcacd6cbe2767b63dca324f4e5e81dc496b69cda061</value>
tx      </key>
tx      <key>
tx        <id>3c31b2</id>
tx        <value>fd272879b21a7443aad17a4f9145dcb405a84cbfd33d9f68443c5318d0e2b74a</value>
tx      </key>
tx      <key>
tx        <id>3c31b3</id>
tx        <value>b8f657bde53cf8cb1910e7e584c48b21c13131979fb1b58ae6854a195543de51</value>
tx      </key>
tx      <key>
tx        <id>3c31b4</id>
tx        <value>79be3298d1cb06613d6bd50b781ca2b9c3c2f01e7c20c13c86b46ca80e3fec53</value>
tx      </key>
tx      <key>
tx        <id>3c31b5</id>
tx        <value>3bac8e4cfa0f5a65b6bb5aaa839f58e98a931c98baf9ce9b66ef5050b3f1a338</value>
tx      </key>
tx      <key>
tx        <id>3c31b6</id>
tx        <value>8a4b80a18d65de4b1034d03fc20552c4954fa74363b45a3bd349909d33723b6c</value>
tx      </key>
tx      <key>
tx        <id>3c31b7</id>
tx        <value>d87c88079d82686287baee769223552daa01eafda9ab9c2a1a49e8b659cb3c37</value>
tx      </key>
tx      <key>
tx        <id>3c31b8</id>
tx        <value>3e42360fc8e4f901ee669fa666c246c8717c4cc8b7b8152a410ba954f5914d28</value>
tx      </key>
tx      <key>
tx        <id>3c31b9</id>
tx        <value>5fc4d0529c9b7ab6ab085530c452eea693efe439ac2aeddc6c73734b87c8104e</value>
tx      </key>
tx      <key>
tx        <id>3c31ba</id>
tx        <value>fb5e8976122084e250d4c31b9cd4c14fcfaf9e7866e8403bb8fe097bab75e81a</value>
tx      </key>
tx      <key>
tx        <id>3c31bb</id>
tx        <value>880ed292a1a33de70483496af17cdbec8998cf1fdd74c8f6030653573532b810</value>
tx      </key>
tx      <key>
tx        <id>3c31bc</id>
tx        <value>d051ba1d1108f1adb90e86e32269e000719ad6658e14b3b03e9b6b5581a01d44</value>
tx      </key>
tx      <key>
tx        <id>3c31bd</id>
tx        <value>7380fc89bbbce8fbbda53b85fc69757f48744596a9d93f181720696f36bf3374</value>
tx      </key>
tx      <key>
tx        <id>3c31be</id>
tx        <value>a5d537f462a035c828c4749a8894467c96e93d7cf0a8567241661309b371da2e</value>
tx      </key>
tx      <key>
tx        <id>3c31bf</id>
tx        <value>53651ad60223b76a5470aa56f4465ca854653b85b3741aebe7de21dbbf899410</value>
tx      </key>
tx      <key>
tx        <id>3c31c0</id>
tx        <value>10da8409f21c268f00e198d38219afbdfe0d7c771f6b3c3d8e20936b8cbd9933</value>
tx      </key>
tx      <key>
tx        <id>3c31c1</id>
tx        <value>3fb983148a0a6deac0fe7a2fdddbd0dd7b94ccec6f6b55fcd3ea5448ead06119</value>
tx      </key>
tx      <key>
tx        <id>3c31c2</id>
tx        <value>df9bac26ab57194e6c2d906f2bc47afdb96ba10a90ebd564146ced25a3966363</value>
tx      </key>
tx      <key>
tx        <id>3c31c3</id>
tx        <value>7942706ed14d934d5b5090b977930f99be0da0adbc79bf4630394db6b4b57b30</value>
tx      </key>
tx      <key>
tx        <id>3c31c4</id>
tx        <value>b2a2d9cf45701ab4c78e5cccd6d46ff972e6b0fc7d61d4b48dc8b3fd52807b58</value>
tx      </key>
tx      <key>
tx        <id>3c31c5</id>
tx        <value>97e70b318ed4aa0dde8ae1703a1dd7f51cc8680551963d27f605df87e58e347a</value>
tx      </key>
tx      <key>
tx        <id>3c31c6</id>
tx        <value>cd9568c16b87bbc42ea7421cc501c28b10d4acf61a18ff1fefd5b0f1d8368a7f</value>
tx      </key>
tx      <key>
tx        <id>3c31c7</id>
tx        <value>74a35d07bd5668f3fae8309a573c21809e5e6e5dde9a4894f6fa8f1ccc4d486c</value>
tx      </key>
tx      <key>
tx        <id>3c31c8</id>
tx        <value>85c495f5845d44cc063a18eaa841e30bb7604c0b7532245ee2c58ee76e29f555</value>
tx      </key>
tx      <key>
tx        <id>3c31c9</id>
tx        <value>f3d7bd7fdb268d88e2b7e8e1752ba36711a340706657b965dabf36f88b48bb30</value>
tx      </key>
tx      <key>
tx        <id>3c31ca</id>
tx        <value>39096ccf5eb95031e6e329b6936ecdee7d4c52910fb5231c6b0cb6c02e34955f</value>
tx      </key>
tx      <key>
tx        <id>3c31cb</id>
tx        <value>5d3846c6c1aa72c19090324609a1ce235ec515eebcd460c4eb043f10a5418d75</value>
tx      </key>
tx      <key>
tx        <id>3c31cc</id>
tx        <value>e7877b50e7a5cc650c6a6137f62d85dd23a56c23b98d9d2f164b6e703ec02271</value>
tx      </key>
tx      <key>
tx        <id>3c31cd</id>
tx        <value>553ab1ff643c26b5c90457ecfccdcadb5bab4335bc2f45ee82b69a051c51c03d</value>
tx      </key>
tx      <key>
tx        <id>3c31ce</id>
tx        <value>ca0866090fd7a8230fee846f5d07aec4005e1052c60a8ee9ef3d9003b9e4ff6c</value>
tx      </key>
tx      <key>
tx        <id>3c31cf</id>
tx        <value>3a8a8c8fb32b26b366a9f4051a56f81e7e49448a02a5a681b892ace9d8f7ca24</value>
tx      </key>
tx      <key>
tx        <id>3c31d0</id>
tx        <value>00152e9397110f6346ba0e5492624271914cf6c0502e3b65f24e0f1613d47a0e</value>
tx      </key>
tx      <key>
tx        <id>3c31d1</id>
tx        <value>ff15915953e46487fc9431b90b4a3f5ed882df0a77654d6f31a67c58ce299d61</value>
tx      </key>
tx      <key>
tx        <id>3c31d2</id>
tx        <value>13917131d4d6319a22a7d6331def4c7ac7f5eb54f95f9f4c6bb127991604f912</value>
tx      </key>
tx      <key>
tx        <id>3c31d3</id>
tx        <value>03a9144c164ea2f4226bd368784ec62abab39f2d534e109a89f6c690742cc55e</value>
tx      </key>
tx      <key>
tx        <id>3c31d4</id>
tx        <value>8e0d29b30aaeb650623e7c83dbfc9d61ddbcddf74bdca6b14185e85f6ea0e87e</value>
tx      </key>
tx      <key>
tx        <id>3c31d5</id>
tx        <value>a9c1152ea937275052659d2200ce7c1e17424aaf490744bc602e88894440cc18</value>
tx      </key>
tx      <key>
tx        <id>3c31d6</id>
tx        <value>746d0f4b9d9385b23c3f249e697275475134097bd9ea90d6f1d790ff5e82431c</value>
tx      </key>
tx      <key>
tx        <id>3c31d7</id>
tx        <value>2f9926c0c9a352b137ddb3ffc9bceca87cb6483e88396683dba73501c09d7346</value>
tx      </key>
tx      <key>
tx        <id>3c31d8</id>
tx        <value>d8586e9308333f4ad9758b2f7ef050a73b5df9a6db54dba6f10363b0710fa359</value>
tx      </key>
tx      <key>
tx        <id>3c31d9</id>
tx        <value>673749c445228488658950539465b79663cc0f7d8faa89d1574ddc02dd1a442e</value>
tx      </key>
tx      <key>
tx        <id>3c31da</id>
tx        <value>38c77b9e8139ac3c6fd1482ea35d35a6b05e047b5f60266dbc08ddf3f245d615</value>
tx      </key>
tx      <key>
tx        <id>3c31db</id>
tx        <value>56d9175baed88ac12b82a2cd8f7994040949c39c9f95a59c1482e7d63e24af12</value>
tx      </key>
tx      <key>
tx        <id>3c31dc</id>
tx        <value>efaea76ba9abefb8b06699f76f43782dd60ca602657c3ce919fbe0b857a1f028</value>
tx      </key>
tx      <key>
tx        <id>3c31dd</id>
tx        <value>37c3c1df9eec290b293c07e25d218e5315b72147f20a1976c45c40de79b7f803</value>
tx      </key>
tx      <key>
tx        <id>3c31de</id>
tx        <value>371a8e19db17ab7f598848ec54abe5cf59dd729ab43df0219fe9e8d8efcd4c78</value>
tx      </key>
tx      <key>
tx        <id>3c31df</id>
tx        <value>9be052bd11189826aa2f3abef23fc428d7174e9889e0a54482d1b8461e2f0f7b</value>
tx      </key>
tx      <key>
tx        <id>3c31e0</id>
tx        <value>8b80fe916fa54658c33272e55f38ff432759b763f7b48abd295206988a3f6507</value>
tx      </key>
tx      <key>
tx        <id>3c31e1</id>
tx        <value>4f332e56102d8d43d259dd7b90fe276421594cc80e5a7e33e2d01d0093694e67</value>
tx      </key>
tx      <key>
tx        <id>3c31e2</id>
tx        <value>ca9de878f3ebc654cd2fd84902bf86c27e0ac4588e12ec6cc5c785c0aebbb061</value>
tx      </key>
tx      <key>
tx        <id>3c31e3</id>
tx        <value>e7bb22991ac3ecca562ac1cfe3d4e61ecedfb5a6cd5ab980f3d8a5516be40d2b</value>
tx      </key>
tx      <key>
tx        <id>3c31e4</id>
tx        <value>8c638193e7ecf895a79e120442efdf61362049d250e830cbdf82f2df29660a44</value>
tx      </key>
tx      <key>
tx        <id>3c31e5</id>
tx        <value>bd75d4d7fe7f305dc9a08d840e01fff827790ea6dd94d6671fae42c7ae854115</value>
tx      </key>
tx      <key>
tx        <id>3c31e6</id>
tx        <value>58f8e09e4cfbec52fd87eea7c8257bb7cdca82cefdf4524aff3634c4ad292f64</value>
tx      </key>
tx      <key>
tx        <id>3c31e7</id>
tx        <value>b8df19625c80a0db14220f843bc243654a4518895424ed4858edb1ff1c461253</value>
tx      </key>
tx      <key>
tx        <id>3c31e8</id>
tx        <value>162030710db2010ba2b822f3f05bd606c7fbbfb64949082aaf969fee1386321b</value>
tx      </key>
tx      <key>
tx        <id>3c31e9</id>
tx        <value>245d1701df4e69afab8ba29ebf93043b99474ba0aa8838eb3e67887867949839</value>
tx      </key>
tx      <key>
tx        <id>3c31ea</id>
tx        <value>bac25abc3a42ef875786adaa265956e062aa04e01034aed706cbdc48291c1066</value>
tx      </key>
tx      <key>
tx        <id>3c31eb</id>
tx        <value>833d55e7378946b02846168c2060dff407c81bba6a03bb145addfa73c3dc045e</value>
tx      </key>
tx      <key>
tx        <id>3c31ec</id>
tx        <value>a250bc2fcd2d12155bf0774e017e56f27f68c3ba41671894663aceed32d36b5a</value>
tx      </key>
tx      <key>
tx        <id>3c31ed</id>
tx        <value>520fe95ccec0385406236f1617dccbb873e96d28324189fe9ad9bd70549b8c6b</value>
tx      </key>
tx      <key>
tx        <id>3c31ee</id>
tx        <value>26307b06ea183a66668aa372e9869b2a2b2fd5f440d4f96eca3ab369e221a701</value>
tx      </key>
tx      <key>
tx        <id>3c31ef</id>
tx        <value>831ba609833207d48cd98dd793c90406b3efb6b18b1f037f4bed81ed901f8505</value>
tx      </key>
tx      <key>
tx        <id>3c31f0</id>
tx        <value>7eb677bd6c9e3ea32e8108896704cc36df28ab170c060fd9c764a55750c6de74</value>
tx      </key>
tx      <key>
tx        <id>3c31f1</id>
tx        <value>275be9e2d092cb53c7021355339439a19106229b2cd130c40fa6e08bd6ba7526</value>
tx      </key>
tx      <key>
tx        <id>3c31f2</id>
tx        <value>a85d0eea63c4e1fb24ca2f4dfc3a6807f87fb84cd20914485518fcbd4374bf04</value>
tx      </key>
tx      <key>
tx        <id>3c31f3</id>
tx        <value>c067c5e690749c50d70f65ca7235d45d555cf6a5ef0b0598060ad6e698160e40</value>
tx      </key>
tx      <key>
tx        <id>3c31f4</id>
tx        <value>53491d513741c43525fb98f62586ceaa748697635ed727dedfd690248d85554e</value>
tx      </key>
tx      <key>
tx        <id>3c31f5</id>
tx        <value>c3a6910f1ea52172b1e36a0e0df19f7e3b3dd4604a0ebb2ff848f59ded4b875e</value>
tx      </key>
tx      <key>
tx        <id>3c31f6</id>
tx        <value>bfd65fa880beb2dd57fe7d345d9bb40c3150fc63c7a33fc7c05d386018481d12</value>
tx      </key>
tx      <key>
tx        <id>3c31f7</id>
tx        <value>26b94a0a0c95c2ad24a0d91e947584489e641510db027832d77d24fa60c77e58</value>
tx      </key>
tx      <key>
tx        <id>3c31f8</id>
tx        <value>81984a5b2963716098b84736f6ecec1ce68b3361c371aac426b40f3facb1bf4b</value>
tx      </key>
tx      <key>
tx        <id>3c31f9</id>
tx        <value>904b86e94866b68f8b59eb02b172eafe6b2edce5b905274540ff4000869ba20b</value>
tx      </key>
tx      <key>
tx        <id>3c31fa</id>
tx        <value>1834d9a8d582199e23324a8d27b4d3a911c2b27c1cefd499f5ddf596107b6203</value>
tx      </key>
tx      <key>
tx        <id>3c31fb</id>
tx        <value>7ea18952104561a532f09819fe7d381960f11e09ff4887802fcfdda76a15d719</value>
tx      </key>
tx      <key>
tx        <id>3c31fc</id>
tx        <value>cd41343ce7d5cac71181908d8f1ec9cfb3648ac706992f2cad55e70d51023628</value>
tx      </key>
tx      <key>
tx        <id>3c31fd</id>
tx        <value>d1b0580a4daf2e224e87ef4b1093ace81aa54b2783c4e8943ab4ba7f97649a4f</value>
tx      </key>
tx      <key>
tx        <id>3c31fe</id>
tx        <value>aa5b236de264030c40d74055d58943c91706a419bc55eabaf9098b7735814a0b</value>
tx      </key>
tx      <key>
tx        <id>3c31ff</id>
tx        <value>2eb5ce278f8192d6f574411d086585878bc7f13f4a93b7dfdd764feb7acce608</value>
tx      </key>
tx      <key>
tx        <id>3c3200</id>
tx        <value>0315f0589d997bf59fb443e7d05fea3c0c876066ee54e27d1ec5d689e5432d0b</value>
tx      </key>
tx      <key>
tx        <id>3c3201</id>
tx        <value>68f206236b6a8c9577231a9271bc9400d7491d45a3262526490785abc295ce61</value>
tx      </key>
tx      <key>
tx        <id>3c3202</id>
tx        <value>d6170dfc1c3b1e0114a45579cbc0ded168086cca8b66ff5abd237ed08f15b920</value>
tx      </key>
tx      <key>
tx        <id>3c3203</id>
tx        <value>3842a4690ee22077b6e9e900a96106594ce1e73e4e08c97bdb460176ff116048</value>
tx      </key>
tx      <key>
tx        <id>3c3204</id>
tx        <value>3dbd29f0ebd23b772ea0969076c5ec2979f159f4df896b161b8c7977db7e302b</value>
tx      </key>
tx      <key>
tx        <id>3c3205</id>
tx        <value>59f688f3e376af333455e1ccf24f167deaeeef3f25961e19e3968047dafed151</value>
tx      </key>
tx      <key>
tx        <id>3c3206</id>
tx        <value>c0f1f007aa36c53fd24fedbb8e9f51575d8f36a003cd456730191f225d6f5f43</value>
tx      </key>
tx      <key>
tx        <id>3c3207</id>
tx        <value>162480e0effa4a6039889cac21d5f6de175e54a8ee4ca530f61ed0ee6af4d265</value>
tx      </key>
tx      <key>
tx        <id>3c3208</id>
tx        <value>8e7ec708e59d735ac45f4eff49fa7ce227a31a91fa9ca7735f6ca0a1f98b8573</value>
tx      </key>
tx      <key>
tx        <id>3c3209</id>
tx        <value>fa1a9946b27874a4508eb555a0682fd52e6c4ee509b55a06719a31c8746a4d73</value>
tx      </key>
tx      <key>
tx        <id>3c320a</id>
tx        <value>b580e92ba699cdc5240742b9fcad5465dd2a0a4fbc47c489b7f6a285d299d016</value>
tx      </key>
tx      <key>
tx        <id>3c320b</id>
tx        <value>9d1f666eccd4b73c2207bedf36a7ff07467e5ae4a7b2cd40496c65ee23a0712e</value>
tx      </key>
tx      <key>
tx        <id>3c320c</id>
tx        <value>ef65d2b4dbf1bdb99fd568d23ac9db0bbf34951275fdb6a4db542fed2dcfcc37</value>
tx      </key>
tx      <key>
tx        <id>3c320d</id>
tx        <value>bef8c921cff5196fb25c33aafa9c71a9b1fb43c2317d139caea3486088cbe155</value>
tx      </key>
tx      <key>
tx        <id>3c320e</id>
tx        <value>11b4585b7a9b1f71a3778ffa423bf2d736a974a531aa3efe8dc16d2873aa7149</value>
tx      </key>
tx      <key>
tx        <id>3c320f</id>
tx        <value>48302394c004d6a6853f241256cb8b98898e1aa705fc71c5ffdd625956b0425e</value>
tx      </key>
tx      <key>
tx        <id>3c3210</id>
tx        <value>3a24fecee04dee6f52d11c7efe5d94fd8d0604a9c5166311a4f41360e3244173</value>
tx      </key>
tx      <key>
tx        <id>3c3211</id>
tx        <value>7e46c757732b9bca7883114238c1948c3572e38d8cf3d092e7a9a167679ca95b</value>
tx      </key>
tx      <key>
tx        <id>3c3212</id>
tx        <value>a59ba459b5f080f85579047a05172d96e0797e5b693ada3eb574b0393689c708</value>
tx      </key>
tx      <key>
tx        <id>3c3213</id>
tx        <value>3f12a90f05a81731df3150414332504a172dd7269cd7388555a9e296f644be26</value>
tx      </key>
tx      <key>
tx        <id>3c3214</id>
tx        <value>3da8fb1a32dc49240a676e9e4276cb08f1544c229a64ee8ae7839eb91e4ef43e</value>
tx      </key>
tx      <key>
tx        <id>3c3215</id>
tx        <value>4d83e6f68d79f290075b869185e2612f0a3aab4a20b77c4f7b2aa8a19348376e</value>
tx      </key>
tx      <key>
tx        <id>3c3216</id>
tx        <value>851b4cc19f9561f868cd047adc60cf9b2a5e0be88b4b0b10bbac315ef89ae860</value>
tx      </key>
tx      <key>
tx        <id>3c3217</id>
tx        <value>86ad03165c053992a08d496eb9ed7de0fd363f57edec4f3bf213c7373cc8e351</value>
tx      </key>
tx      <key>
tx        <id>3c3218</id>
tx        <value>a01915d3bc9dbb2464a54a03d9f81456d267e3eb96ca651e6c9a94933ce74e29</value>
tx      </key>
tx      <key>
tx        <id>3c3219</id>
tx        <value>15091e263c543bb2bf5a39669ab45dc141f4933693099b11f7e19e22175acd78</value>
tx      </key>
tx      <key>
tx        <id>3c321a</id>
tx        <value>a7350061fd0b9d02405fcb2f0d477ea4246cca3d99e5671795def1f12071d25d</value>
tx      </key>
tx      <key>
tx        <id>3c321b</id>
tx        <value>9c8af785bbc8aa1742fc2d326398abd2de25839bd20cfa967ab91764cc9dc123</value>
tx      </key>
tx      <key>
tx        <id>3c321c</id>
tx        <value>f242ac5a330e3c46852bbb3e9c94af104a85a18900e34ef89e941a35bd13620a</value>
tx      </key>
tx      <key>
tx        <id>3c321d</id>
tx        <value>66d7337412043c1707283b9647f8bebd4290c5fa2b9da26bced999dd81f43507</value>
tx      </key>
tx      <key>
tx        <id>3c321e</id>
tx        <value>c85cd5b12f46f410db3c1b5a0bd36fc28d580ac689830dffba6967b15d8e441c</value>
tx      </key>
tx      <key>
tx        <id>3c321f</id>
tx        <value>56f59fd19d4cec62b0b9a7bec9adf05aef44ea58d2fa09597f5daf0620fe3c78</value>
tx      </key>
tx      <key>
tx        <id><2 </id>
tx        <value>ae9a575603dee6d9f9e013269aa6d275d575122b42a0ee778ae3e5edeaf5407c</value>
tx      </key>
tx      <key>
tx        <id><2!</id>
tx        <value>3bdcd49259b98bd9c7e66bead20b90b5a7badd7c57cf46cf1e6212ca824f8a5a</value>
tx      </key>
tx      <key>
tx        <id><2"</id>
tx        <value>6611a5510ef4a66c431caa6427b4d4cbbc0a05bbc1410dd1dfc59e26c71d7c63</value>
tx      </key>
tx      <key>
tx        <id><2#</id>
tx        <value>96e4d1990906988da6292015106acd3e5e516e78e8432662f8baf33ffcc5427f</value>
tx      </key>
tx      <key>
tx        <id><2$</id>
tx        <value>b66c4f736f64e44386236df5d49bb69437d4adbd28743e5bf9ac1412ce75b654</value>
tx      </key>
tx      <key>
tx        <id><2%</id>
tx        <value>03266b2917cfb65fed6869bf10a28790c590c47f93ba150fd84fcd87bfb4431a</value>
tx      </key>
tx      <key>
tx        <id><2&</id>
tx        <value>b667c098c12b8b255731d54a465da1f59bd6f6b41a5e9f0da47d544b48757664</value>
tx      </key>
tx      <key>
tx        <id><2'</id>
tx        <value>c74b51934efae96648736541e591b66e85a87bc3f9e2612ce85844d30f7b3222</value>
tx      </key>
tx      <key>
tx        <id><2(</id>
tx        <value>729811bfdda63d50cbc4b1b23e387467f7b67470d9c04f161e296cf7fdec6818</value>
tx      </key>
tx      <key>
tx        <id><2)</id>
tx        <value>e87a28c2174faf302374f2df03a2e1c365a9fbd3fd74e55df153fe50bc7c8c4a</value>
tx      </key>
tx      <key>
tx        <id><2*</id>
tx        <value>1fd3e086972be839457571d1d79f528ddfcdd36e5632fb08a9b1bf4de93cbd12</value>
tx      </key>
tx      <key>
tx        <id><2+</id>
tx        <value>f24ac20bd4946ce908eec643259486f73c961b824690587e33cc5d5905a16c4a</value>
tx      </key>
tx      <key>
tx        <id><2,</id>
tx        <value>17f905aaa3966e62b23b873c319b442a73051fb0d5677987e0bc1e06bd664941</value>
tx      </key>
tx      <key>
tx        <id><2-</id>
tx        <value>59f6f173f1a426edffc4e824b34c82bc005682e2d5fb7abfd300bd6b6d40bd04</value>
tx      </key>
tx      <key>
tx        <id><2.</id>
tx        <value>e14134a56abe6c30f2654c6d7beb67fd02262762cbe36a55f0c6b42aed13522e</value>
tx      </key>
tx      <key>
tx        <id><2/</id>
tx        <value>a154359a11990f69dcf8fc24b8f2dc11ad97076100bacae1e73b71e781f4c649</value>
tx      </key>
tx      <key>
tx        <id><20</id>
tx        <value>0ba7b8cd5d42ec1dbaa3954c2c9bcb9238e7d983374c4f8efc02f4c8ed8b6e65</value>
tx      </key>
tx      <key>
tx        <id><21</id>
tx        <value>625ceda9aba74c6f99fbc553946c1b0222f62cacfdd150a97701ae7d2f459954</value>
tx      </key>
tx      <key>
tx        <id><22</id>
tx        <value>30f69ab3420904e0862a22bb4b48bbeade4f3d4141e65cb550804654d9aa3b4d</value>
tx      </key>
tx      <key>
tx        <id><23</id>
tx        <value>7b1ae17dda115adaf7b4440dc11977088d30f907a4140016fb74d020c47b042d</value>
tx      </key>
tx      <key>
tx        <id><24</id>
tx        <value>19abea50f2c53e89a7640e2243e723a877da435cbc6c676426d59d679fc1364a</value>
tx      </key>
tx      <key>
tx        <id><25</id>
tx        <value>4aee8e18799c2e21570e92b9a661bf135bf9902d21552132b94f4497a9221449</value>
tx      </key>
tx      <key>
tx        <id><26</id>
tx        <value>83f7098a87e0dbdbf6ef935f074f7a9b3780e33a2b9e9b2748db83dc63d4725c</value>
tx      </key>
tx      <key>
tx        <id><27</id>
tx        <value>82c5f43ccd7976c1819e8d79dde40bf1b0688ea6b1c1ea6202effad9585e0d5f</value>
tx      </key>
tx      <key>
tx        <id><28</id>
tx        <value>f1c907981951ad7b257447e6bd38e7407e7a67c9756219d06063060a4d18c34f</value>
tx      </key>
tx      <key>
tx        <id><29</id>
tx        <value>8ad25a8b6a16c0df11530346d06a08c70214b0fa5767ebbee04ee4dc20b0dc1a</value>
tx      </key>
tx      <key>
tx        <id><2:</id>
tx        <value>1c3a5c4a37b05d4dc57cf4acb27cb49892e36653e593bdca3a29d63e61e44e20</value>
tx      </key>
tx      <key>
tx        <id><2;</id>
tx        <value>4b5e429ca2063cc786b559f0519e95c00a9b8e35b4ac494664f9aad87174c22f</value>
tx      </key>
tx      <key>
tx        <id><2<</id>
tx        <value>0139ceb9cf53d929bfa6e3abbdef418a2faf221db966ad657985d4f2a0aeba0e</value>
tx      </key>
tx      <key>
tx        <id><2=</id>
tx        <value>6400b0aab95643f1031432f46a7203c226414c97f3013e723ff3e3e5fa718a4a</value>
tx      </key>
tx      <key>
tx        <id><2></id>
tx        <value>e8f20da398388e8adf576e30b9f06b67ed8c39b5321ecbcb85e6bf8cc646550f</value>
tx      </key>
tx      <key>
tx        <id><2?</id>
tx        <value>35770d2fc903a2b2e21868ab69a4e90b8f825a55f66c520ac19db5d1c0f83957</value>
tx      </key>
tx      <key>
tx        <id><2@</id>
tx        <value>51e35f69dfd047776483c7800fd87ea71c59752d45834eaf68f91fc507291b6b</value>
tx      </key>
tx      <key>
tx        <id><2A</id>
tx        <value>2a54e977882ffa47430afc45fa1c4125bee92c3d7a05c8f82d995ce56c7a3c70</value>
tx      </key>
tx      <key>
tx        <id><2B</id>
tx        <value>f18eea0e5d4c030cfdb95dfefbefc8a87ec85cf3b34170155446b854df22ed48</value>
tx      </key>
tx      <key>
tx        <id><2C</id>
tx        <value>60410930058cd690960f7c6c999c3128784f759bed0cbd908e7cfb32372ec004</value>
tx      </key>
tx      <key>
tx        <id><2D</id>
tx        <value>eda677d1f0eaf4f4132cbd2d7cae64f9fc5f082a9e5fe1b5e0d9957277c01e17</value>
tx      </key>
tx      <key>
tx        <id><2E</id>
tx        <value>aa89fe8d1001a3ba9cb1549f7d7dbdb8e23db373884361955b536422100ee92d</value>
tx      </key>
tx      <key>
tx        <id><2F</id>
tx        <value>a96820ff1749c4c3712a63079a0c4f708093710c6f605b596ce62a02cb3c7e73</value>
tx      </key>
tx      <key>
tx        <id><2G</id>
tx        <value>bfe6ace5c6df0837512fb47972929013d82ac9f60a205901e3a387c0f24bdf21</value>
tx      </key>
tx      <key>
tx        <id><2H</id>
tx        <value>db294aa6fa2389a4ead9a07f1897605275e9706b7649b887e0db5ea614178970</value>
tx      </key>
tx      <key>
tx        <id><2I</id>
tx        <value>e8cece7270c8e32b1f725c306cdbe57222ab4ac9974dfa6df32043cc1a05b165</value>
tx      </key>
tx    </list>
tx    <skey>
tx      <id>0051d1</id>
tx      <value>f94ba2475bba6f6a8f33446e30d5f3f3668cf061bb03ce950f06d180e2911c5c</value>
tx      <signature>28636ec6e9b9f93d49cfd0bfd2efb19916f4c4b3efaa1f6bd9e735eae065a921302d1e94e890dd1c55217a4c29c630d4310920246a0b9e4b7d97edb509792a07</signature>
tx    </skey>
tx  </iq>

rx  <notification from="xxxxxxxxxxx42@s.whatsapp.net" type="contacts" id="2294069700" offline="0" t="1465052238">
rx    <update jid="xxxxxxxxxxx42@s.whatsapp.net"></update>
rx  </notification>

tx  <iq id="04" xmlns="urn:xmpp:whatsapp:sync" type="get">
tx    <sync mode="delta" context="interactive" sid="sync_sid_full_61f33788-c3e8-46b9-8de0-8be97d736a71" index="0" last="true">
tx      <user>+xxxxxxxxxxx42</user>
tx    </sync>
tx  </iq>

rx  <ib from="s.whatsapp.net">
rx    <offline count="1"></offline>
rx  </ib>

rx  <presence from="xxxxxxxxxxx49@s.whatsapp.net"></presence>

rx  <iq from="xxxxxxxxxxx49@s.whatsapp.net" type="error" id="1">
rx    <error code="404" text="item-not-found"></error>
rx  </iq>

rx  <iq from="s.whatsapp.net" type="error" id="02">
rx    <error code="404" text="item-not-found"></error>
rx  </iq>

tx  <iq id="05" type="set" xmlns="urn:xmpp:whatsapp:push" to="s.whatsapp.net">
tx    <config platform="gcm" id="HAKkf_RfJcP:APA91b4cbYmToILBjdTTMI2yHYROB7xFinn8fQzlqYTC9yhOlDnssRnSX1yeNeZCyRi8Tg_5JQQt-GNsQHmiN257t_sLfXL6Quv_sjcELI9u2Oz_9_Sk42oVjAkOz095McfQSmL9xqMG"></config>
tx  </iq>

rx  <iq from="s.whatsapp.net" type="result" id="03"></iq>

rx  <iq from="s.whatsapp.net" type="result" id="05"></iq>

rx  <iq from="xxxxxxxxxxx49@s.whatsapp.net" type="result" id="04">
rx    <sync index="0" sid="sync_sid_full_61f33788-c3e8-46b9-8de0-8be97d736a71" last="true" version="1465052117040485">
rx      <in>
rx        <user jid="xxxxxxxxxxx42@s.whatsapp.net">+xxxxxxxxxxx42</user>
rx      </in>
rx    </sync>
rx  </iq>

tx  <presence type="unsubscribe" to="+xxxxxxxxxxx42@s.whatsapp.net"></presence>

tx  <presence type="subscribe" to="+xxxxxxxxxxx42@s.whatsapp.net"></presence>

------- Socket Disconnection -------
Fatal Error, script ended.
------- Restarting the Script -------

tx  <stream:features></stream:features>

tx  <auth user="xxxxxxxxxxx49" mechanism="WAUTH-2"></auth>

rx  <from s.whatsapp.net=""></from>

rx  <stream:features></stream:features>

rx  <challenge>51e21a3c84db48bf0d52476e3fd138fd3a4f8765</challenge>

tx  <response>6f567face4d05a62185f4130a773c812be7f2adb552a5fd8bff6b5aba951f107550221642f486e00726c038bb8312ed6f87d9d59ba30a25c26ea6d7104ce1f55d4a3951638b6a7a1133825417e4d9f</response>

rx  <failure>
rx    <not-authorized></not-authorized>
rx  </failure>

```

**Comment by Sdlion** [comment]
I haven't realized before, but from one key member of the list made by `sendClientConfig()` on onwards, the ID is just wrong. It breaks the tags.

Line 665
```xml
tx      <key>
tx        <id>3c321f</id>
tx        <value>56f59fd19d4cec62b0b9a7bec9adf05aef44ea58d2fa09597f5daf0620fe3c78</value>
tx      </key>
tx      <key>
tx        <id><2 </id>    <!---- THIS ---->
tx        <value>ae9a575603dee6d9f9e013269aa6d275d575122b42a0ee778ae3e5edeaf5407c</value>
tx      </key>
tx      <key>
tx        <id><2!</id>
tx        <value>3bdcd49259b98bd9c7e66bead20b90b5a7badd7c57cf46cf1e6212ca824f8a5a</value>
tx      </key>
tx      <key>
tx        <id><2"</id>
tx        <value>6611a5510ef4a66c431caa6427b4d4cbbc0a05bbc1410dd1dfc59e26c71d7c63</value>
tx      </key>
tx      <key>
(...)
```

Though I can't be sure if this was due to some `screen` character interpretation or if it was the API error.

**Comment by mgp25** [comment]
1) Wrong ids after reconnection
2) You are sending android config, WRONG!
3) sendGetServerProperties only once a week
4) sendSync time to time, i have seen 2 syncs frequently

...

It could to these things i tell you. Also, if you don't expect anything to receive, dont reconnect

Regards

**Comment by Sdlion** [comment]
Thanks, I didn't get that second sync in the log. I'll check my code, it's weird.

Could you elaborate about that I'm sending "android config"?
Is from `sendClientConfig()` ?
I used it since it was on the `Simple CLI client.php` 
I could open up a pull request deleting that line to avoid future mistakes for other people.
(And translate the spanish comments adding a note about `sendGetServerProperties()` that shouldn't be used more than once in a week)
